src/vm/loaderallocator.cpp

```diff
@@ -546,7 +546,8 @@ BOOL QCALLTYPE LoaderAllocator::Destroy(QCall::LoaderAllocatorHandle pLoaderAllo
 
     BEGIN_QCALL;
 
-    if (ObjectHandleIsNull(pLoaderAllocator->GetLoaderAllocatorObjectHandle()))
+    OBJECTHANDLE objHandle = pLoaderAllocator->GetLoaderAllocatorObjectHandle();
+    if (*(Object**)objHandle == NULL)
```

Comment:
I think it is better to have it in gchandleutilities than to have casts that violate the abstraction in random places, even if it is just two places.